test/IDE/complete_call_pattern_after_dot.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %batch-code-completion
+
+struct Foo {
+  func bar() -> Int { 1 }
+}
+let x = Foo().bar
+x.#^AFTER_DOT^#
+// AFTER_DOT: Pattern/CurrModule/Flair[ArgLabels]/Erase[1]: ()[#Int#]; name=()
+
+x. #^AFTER_DOT_AND_SPACE^#
+// AFTER_DOT_AND_SPACE: Pattern/CurrModule/Flair[ArgLabels]/Erase[2]: ()[#Int#]; name=()
```

Comment:
Test case for `Foo().bar.#COMPLETE%#` might be also valuable.

Also, I believe we already have a test case for it, but the method could have arguments with labels. To show the difference between:
```swift
let x = Foo().bar
x.#^COMPLETE^#.        // -> (<#arg#>)
Foo().bar.#^COMPLETE^# // -> (label: <#arg#>)
```

Reply:
Seems to work. Added test cases.